ambry-tools/src/main/java/com.github.ambry/store/DumpIndexTool.java

```diff
@@ -541,20 +705,23 @@ public IndexProcessingResults processIndex(File replicaDir, Set<StoreKey> filter
   }
 
   /**
-   * Will examine the index files in the File replicas for store keys
-   * and will then convert these store keys and return the
-   * conversion map
-   * @param replicas replicas that should contain index files that will be
-   *                 parsed
+   * Will examine the index files in the File replicas for store keys and will then convert these store keys and return
+   * the conversion map
+   * @param replicas replicas that should contain index files that will be parsed
    * @return conversion map of old store keys and new store keys
    * @throws Exception
    */
-  public Map<StoreKey, StoreKey> retrieveAndMaybeConvertBlobIds(File[] replicas) throws Exception {
-    Pair<Boolean, Map<File, DumpIndexTool.IndexProcessingResults>> resultsByReplica =
-        getIndexProcessingResults(replicas, null);
-    boolean success = resultsByReplica.getFirst();
+  public Map<StoreKey, StoreKey> retrieveAndMaybeConvertStoreKeys(File[] replicas) throws Exception {
```

Comment:
looks like this method is temporarily not used?

Reply:
Yes - not used here but was an API that was added because it was useful in a previous PR